src/Entity/SocialPost.php

```diff
@@ -19,8 +19,8 @@
  *   entity_keys = {
```

Comment:
We might need to do something like in [Social Auth](https://github.com/drupalsocialinitiative/social_auth/pull/6)